src/almanac.js

```diff
@@ -179,7 +178,7 @@ export default class Almanac {
    * Interprets value as either a primitive, or if a fact, retrieves the fact value
    */
   getValue (value) {
-    if (isObjectLike(value) && Object.prototype.hasOwnProperty.call(value, 'fact')) { // value = { fact: 'xyz' }
+    if (value != null && typeof value === 'object' && Object.prototype.hasOwnProperty.call(value, 'fact')) { // value = { fact: 'xyz' }
```

Comment:
I believe these checks could further be reduced to use the instanceof operator which fails for null values.
```suggestion
    if (Boolean(value instanceof Object) && Object.prototype.hasOwnProperty.call(value, 'fact')) { // value = { fact: 'xyz' }
```

Reply:
Thanks for suggestion,

looks good, i even run it against [isObjectLike](https://github.com/Tcdian/Lodash/blob/master/test/isObjectLike.test.ts) tests to be sure it works as before :)

Reply:
I think the instanceof check is going to be slightly slower than the typeof check because it has to follow the prototype chain the whole way back to the root.
I also wonder if there's a way to cause an object to not have Object in its prototype chain, not that we need to code defensively for bad behavior but could you defeat the instanceof check by swapping out the prototype on an instantiated object?

Reply:
Doing some more digging it seems the instanceof check is problematic in some situations. I do think we probably want to stick with the typeof check.

See docs on instanceof here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#description

The biggest issue is if you get an object from a different iframe / worker via a post message handler but there are so many other edge cases that I would say the general advice is to use typeof for the primitive checks.